### PR TITLE
make the extraction lambas in terraform

### LINKED ISSUE #162: Terraform extract lambdas
## Description
Make a terraform file to create the lambdas to extract from the three websites.

## Required Files
- extract_lambdas.tf

## User Story
As an engineer I need to be able to easily create/destroy my AWS services to save time.

### PR DESCRIPTION
**Pull request by annalisev**
Create a terraform script to spin up the lambdas for extraction.

Closes #162 .

### CODE REVIEW

**Comment by annalisev** [comment]
I just realised I can put the alert.py lambda in this function too- I will do this now.

**Review by Eibironke — APPROVED** [review]
Perfect, you even got all the correct timeouts 👍

File: terraform/lambdas.tf
Comment (hhm970) [inline terraform/lambdas.tf:1]: Maybe rename it to reflect the pipelining process?

**Comment by annalisev** [comment]
Renamed now!

**Review by hhm970 — APPROVED** [review]
Looks great to me; everything has been configured consistently!

**Pull request merged**